README.md

```diff
@@ -54,10 +54,10 @@ conda activate dscim-facts-epa
 
 Be sure that all commands and analyses are run from this conda environment.
 
-With the environment set up and active, the next step is downloading the required DSCIM-FACTS-EPA input data into the local directory. From the command line run:
+With the environment set up and active, the next step is downloading the required DSCIM-FACTS-EPA input data into the local directory. Assuming you are in the `dscim-facts-epa/scripts` directory, from the command line run:
```

Comment:
I think it makes more sense for the `input` dir to be in the root of the repo. Or at least not inside of `scripts`. @JMGilbert can you adjust the paths and instructions accordingly?

Reply:
I vaguely remember the EPA wanting the input dir to be in the scripts directory for dscim-epa, but maybe I'm remembering wrong.